### PR TITLE
Created Documentation for seq

### LINKED ISSUE #438: Writing documentation for seq command
I am going to add documentation for the seq command

### PR DESCRIPTION
**Pull request by jsbeharr**
# Advanced HW 5 submission

Closes #438 

Name: Justin Beharry
Uniqname: jsbeharr

### CODE REVIEW

File: _commands/basics/seq.md [hunk 1]
```diff
@@ -3,22 +3,64 @@
...
+2
+3
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+#### `seq [FIRST] [LAST]`
```
Comment (tarunsk) [inline _commands/basics/seq.md]: Looks good, but what about options? Maybe include a small part with useful flags for the command as well.

File: _commands/basics/seq.md [hunk 2]
```diff
@@ -3,22 +3,64 @@
...
+
+~~~ bash
+> for i in $(seq 3)
+> do
+>   echo "i is now $i"
+> done
+i is now 1
+i is now 2
+i is now 3
+~~~
```
Comment (tarunsk) [inline _commands/basics/seq.md]: If possible, find a way to make the distinction between the script and the output produced by the script more clear. You could do this by separating them into different code blocks or by including comments somewhere.

File: _commands/basics/seq.md [hunk 3]
```diff
@@ -3,22 +3,64 @@
...
+10
+~~~
 
 ##### Break it down
+ * The `FIRST` argument is where the sequence will start. 
+ * The `INCREMENT` argument determines how the sequence increments by.
+ * The `LAST` argument is where the sequence will end.
 
-#### Example command
+#### Bash Script
```
Comment (tarunsk) [inline _commands/basics/seq.md]: If possible, find a way to make the distinction between the script and the output produced by the script more clear. You could do this by separating them into different code blocks or by including comments somewhere.

**Comment by jsbeharr** [comment]
I changed and added more of the things you requested to the seq page. Check it out.

**Comment by tarunsk** [comment]
Looks good to me! Don't worry about the failing build - that's a problem on our side. Merging this in!

**Pull request merged**